src/App.js

```diff
@@ -9,34 +8,39 @@ class App extends Component {
     super();
     this.state = {
       movies: [],
-      singleMovie: movieData.singleMovie,
+      singleMovie: {},
       error: ''
     }
   }
-
+  
   componentDidMount() {
     fetch('https://rancid-tomatillos.herokuapp.com/api/v2/movies')
       .then(response => response.json())
       .then(data => {this.setState({movies: data.movies})})
       .catch(error => this.setState({error: error}))
   }
-
-  clickPoster = () => {
+  
+  clickPoster = (id) => {   
+    this.fetchMovie(id);
     document.querySelector('.posters-container').classList.add('hidden');
-    document.querySelector('.movie-background').style.background = `url(${this.state.singleMovie.backdrop_path})`;
-    document.querySelector('.movie-background').style.height = `100vh`;
+  }
+
+  fetchMovie(id) {
+    return fetch(`https://rancid-tomatillos.herokuapp.com/api/v2/movies/${id}`)
+      .then(response => response.json())
+      .then(data => {this.setState({singleMovie: data.movie})})
+      .catch(error => this.setState({error: error}))
   }
 
   clickBackBtn = () => {
     document.querySelector('.posters-container').classList.remove('hidden');
```

Comment:
Hopefully you'll be able to refactor these query selectors out once you refactor for Router 👍